apps/src/applab/designMode.js

```diff
@@ -309,6 +309,38 @@ designMode.updateProperty = function(element, name, value) {
     case 'readonly':
       element.setAttribute('contenteditable', !value);
       break;
+    case 'data-is-default':
+      if (value === true) {
+        //Make this one default
```

Comment:
Super-nit: We generally have a space after the beginning of our comment, i.e `// Make...`